src/commands/payid-sign.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  getDefaultAlgorithm,
+  IdentityKeySigningParams,
+  ServerKeySigningParams,
+  signWithKeys,
+  toKey,
+} from '../verifiable'
+import { convertToVerifiedAddress } from '../verifiable/converters'
+
+import Command from './Command'
+
+/**
+ * Signs the currently loaded PayID PaymentInformation using the loaded signings keys.
+ */
+export default class SignPayIdCommand extends Command {
+  protected async action(): Promise<void> {
+    const info = this.getPaymentInfo()
+    const payId = info.payId
+    if (!payId) {
+      this.vorpal.log('missing payid')
+      return
+    }
+    const signingKeys = this.getSigningKeys()
+    if (signingKeys.length === 0) {
+      this.vorpal.log(`you must generate or load a key before signing`)
+      return
+    }
+
+    const updatedAddresses = info.addresses.map((address) => {
+      const jws = signWithKeys(payId, address, signingKeys)
+      return convertToVerifiedAddress(jws)
+    })
+    const updated = {
+      payId: info.payId,
+      addresses: info.addresses,
+      verifiedAddresses: updatedAddresses,
+    }
```

Comment:
So right now, the CLI only lets you sign an address if it already exists as an unsigned address, is that correct?

I think that's reasonable, but we've talked before about people perhaps wanting to not even serve unsigned addresses at all.

Reply:
I wasn't sure what to do there, whether to blank out the address after signing, or leave them as-is. I don't have a strong opinion.

Reply:
I think @sappenin uploaded both signed and unsigned addresses to his PayID server, which makes sense to me. You _probably_ still want wallets that don't (yet?) support V. PayID to be able to send money to you.